Source/WebCore/html/HTMLDetailsElement.cpp

```diff
@@ -115,7 +116,7 @@ void HTMLDetailsElement::didAddUserAgentShadowRoot(ShadowRoot& root)
     root.appendChild(summarySlot);
 
     m_defaultSlot = HTMLSlotElement::create(slotTag, document());
-    ASSERT(!m_isOpen);
+    ASSERT(attributeWithoutSynchronization(openAttr).isNull());
```

Comment:
`hasAttribute` is more efficient and does the same thing

Reply:
Thanks — switched all